Add React Native direct debugger (#1088)
microsoft/vscode-react-native: doc/debugging.md

    "platform": "wpf"
}
```

> ## Debugging React Native Hermes

Reviewer: ```suggestion
## Debugging React Native apps with Hermes enabled
```

Hermes is the new JavaScript engine optimized for running React Native apps on Android. It improves app performance and decreases app size.

> To enable Hermes you are required to use at least version 0.60.2 of React Native. You also need to edit your `android/app/build.gradle` file and rebuild your Android app. You can find the complete instruction [here](https://facebook.github.io/react-native/docs/hermes/).

Reviewer: ```suggestion
To enable Hermes for your app please follow [this official instructions](https://facebook.github.io/react-native/docs/hermes/).
```

> 

Reviewer: ```suggestion

Debugging apps with Hermes enabled is currently experimental and don't fully support all the features that are available for stable debugger. Please, see [this issue](https://github.com/microsoft/vscode-react-native/issues/1073) for more details about the implementation progress.
```

    "platform": "wpf"
}
```

> ## Debugging React apps with Hermes enabled

Reviewer: ```suggestion
## Debugging React Native apps with Hermes enabled
```

Hermes is the new JavaScript engine optimized for running React Native apps on Android. It improves app performance and decreases app size.

> To enable Hermes for your app please follow [this official instructions](https://facebook.github.io/react-native/docs/hermes/).

Reviewer 2: Please change the line to - 
``Click [here](https://facebook.github.io/react-native/docs/hermes/) to learn more about Hermes.``

> Debugging apps with Hermes enabled is currently experimental and don't fully support all the features that are available for stable debugger. Please, see [this issue](https://github.com/microsoft/vscode-react-native/issues/1073) for more details about the implementation progress.

Reviewer 2: ``Debugging apps with Hermes enabled is currently experimental.  Please, see [this issue](https://github.com/microsoft/vscode-react-native/issues/1073) for current known issues on Hermes support. .``

> If you use Hermes engine, you are required to choose one of the following configurations:

Reviewer 2: ``To debug while using Hermes engine, please choose one of the following debug configurations:``

 - React Native Direct: Debug Android

```
{

>     "name": "Debug Android (Hermes)",

Reviewer 2: ``Debug Android (Hermes) - Experimental``

    "cwd": "${workspaceFolder}",
    "type": "reactnativedirect",
    "request": "launch",
    "platform": "android"
}
```

 - React Native Direct: Attach to packager

 ```
{

>     "name": "Attach to packager (Hermes)",

Reviewer 2: ``Attach to packager (Hermes) - Experimental``
